cli/src/pcluster/resources/imagebuilder/parallelcluster_tag.yaml

```diff
@@ -176,7 +187,17 @@ phases:
               NVIDIA_VERSION="$(get_modinfo "nvidia")"
               add_tag "parallelcluster:nvidia_version" "${NVIDIA_VERSION}"
               append_description "nvidia" "${NVIDIA_VERSION}"
-              CUDA_VERSION="$(cat /usr/local/cuda/version.txt | cut -d' ' -f3)"
+              NVIDIA_ENABLED=$(cat /etc/parallelcluster/image_dna.json | jq -r '.cluster.nvidia.enabled')
```

Comment:
is `NVIDIA_ENABLED` needed?

Reply:
Oh, you're right - that was leftover from the previous way I was doing the check. I will remove it.

Reply:
Fixed.